Scripts/RemediationScripts/Remediate-SetDBForMySQLFlexibleServerMinReqTLSVersion.ps1

```diff
@@ -375,7 +374,28 @@ function Set-SecureTLSVersionForDBForMySQLFlexibleServer {
 
     if ($totalDBForMySQLFSWithNonSecureTLSVersionEnabled -eq 0) {
         Write-Host "No Azure Database for MySQL flexible server(s) found with non-secure TLS version enabled.. Exiting..." -ForegroundColor $([Constants]::MessageType.Warning)
-        Write-Host $([Constants]::DoubleDashLine)	
+        Write-Host $([Constants]::DoubleDashLine)
+        
+        if($AutoRemediation) 
+        {
+            $DBForMySQLFlexibleServerDetails | ForEach-Object {
```

Comment:
No need to this.

Reply:
Removed and logged skipped resources where we are checking the tls version.